### DISCUSSION TITLE
CTD plot set the x-axis limits

### DISCUSSION THREAD

**Discussion by lmm91** [dankelley/oce#2022] [General]
Is it possible to set the x axis limits, this would be useful for cross comparison between multiple stations. In ggplot it would be xlim i think, but i am not sure if it is possible here, also with multiple parameters.

**Comment by dankelley** [comment]
You're right -- multiple parameters make it impossible here.

Please note that the generic `plot()` invocation makes choices as to what to show, and the default has a variety of plot types for which no `xlim` (etc) makes sense.  But for serious work you'll want to be calling e.g.
* `plotProfile()`
* `plotTS()`
and so forth, instead of getting `plot()` to call them.  And these lower-level functions each have ways to control limits.  Use e.g. `?plotProfile` to learn about that.
